### PR TITLE
Feature/passport state

### PR DESCRIPTION
**Pull request by MarkCherepovskyi**

### CODE REVIEW

File: internal/cli/main.go [hunk 1]
```diff
@@ -24,6 +28,12 @@ func Run(args []string) bool {
 
 	runCmd := app.Command("run", "run command")
 	serviceCmd := runCmd.Command("service", "run service") // you can insert custom help
+	votingCmd := runCmd.Command("voting", "run voting")
```
Comment (artemskriabin) [inline internal/cli/main.go:32]: add readme description

File: internal/assets/migrations/001_initial.sql [hunk 2]
```diff
@@ -0,0 +1,14 @@
+-- +migrate Up
+
+CREATE TABLE state (
+    id BIGSERIAL PRIMARY KEY,
+    operation_id CHAR(64) NOT NULL ,
+    tx_hash CHAR(64) NOT NULL,
+    proof CHAR(64) NOT NULL,
+    root CHAR(64) NOT NULL,
```
Comment (artemskriabin) [inline internal/assets/migrations/001_initial.sql]: add rarimo root block number field

File: internal/assets/migrations/001_initial.sql [hunk 3]
```diff
@@ -0,0 +1,14 @@
+-- +migrate Up
+
+CREATE TABLE state (
+    id BIGSERIAL PRIMARY KEY,
+    operation_id CHAR(64) NOT NULL ,
+    tx_hash CHAR(64) NOT NULL,
+    proof CHAR(64) NOT NULL,
```
Comment (artemskriabin) [inline internal/assets/migrations/001_initial.sql]: why char64?

File: internal/cli/main.go (hunk 1 above)
Comment (MarkCherepovskyi) [reply internal/cli/main.go:32]: done

File: internal/assets/migrations/001_initial.sql (hunk 2 above)
Comment (MarkCherepovskyi) [reply internal/assets/migrations/001_initial.sql]: done

File: internal/assets/migrations/001_initial.sql (hunk 3 above)
Comment (MarkCherepovskyi) [reply internal/assets/migrations/001_initial.sql]: fixed

**Review by artemskriabin — DISMISSED** [review]

**Review by artemskriabin — APPROVED** [review]

**Pull request merged**